src/Paket.Core/BindingRedirects.fs

```diff
@@ -50,7 +50,14 @@ let internal setRedirect (doc:XDocument) bindingRedirect =
 
 /// Applies a set of binding redirects to a single configuration file.
 let private applyBindingRedirects bindingRedirects (configFilePath:string) =
-    let config = XDocument.Load(configFilePath, LoadOptions.PreserveWhitespace)
+    let config = 
+        try 
+            XDocument.Load(configFilePath, LoadOptions.PreserveWhitespace)
+        with
+        | :? System.Xml.XmlException as ex ->
+            printfn "Illegal xml in file: %s" configFilePath
```

Comment:
To test this code: Put some empty App.config file to your project and try to convert from nuget to paket.


----- Alkuperäinen viesti -----
Lähettäjä: "Steffen Forkmann" <notifications@github.com>
Lähetetty: ‎27.‎3.‎2015 19:36
Vastaanottaja: "fsprojects/Paket" <Paket@noreply.github.com>
Kopio: "Tuomas Hietanen" <tuomas.hietanen@iki.fi>
Aihe: Re: [Paket] If Paket crashes to some invalid user file: let's displaythe file to user so he/she can fix it. (#730)

In src/Paket.Core/BindingRedirects.fs:
> @@ -50,7 +50,14 @@ let internal setRedirect (doc:XDocument) bindingRedirect =
>  
>  /// Applies a set of binding redirects to a single configuration file.
>  let private applyBindingRedirects bindingRedirects (configFilePath:string) =
> -    let config = XDocument.Load(configFilePath, LoadOptions.PreserveWhitespace)
> +    let config = 
> +        try 
> +            XDocument.Load(configFilePath, LoadOptions.PreserveWhitespace)
> +        with
> +        | :? System.Xml.XmlException as ex ->
> +            printfn "Illegal xml in file: %s" configFilePath
could you please use verbosefn
—
Reply to this email directly or view it on GitHub.